config/mainCfg_TauTau_Legacy2018_SMLimit.cfg

```diff
@@ -0,0 +1,103 @@
+[general]
```

Comment:
The file _mainCfg_TauTau_Legay2018_SMLimit.cfg_ is almost the same as the _mainCfg_TauTau_Legay2018.cfg_, the difference is the killing of some redundancies that in this case are not needed.